docs/asciidoc/faq.adoc

```diff
@@ -102,6 +102,13 @@ logger (`_reactor.netty.channel.LeakDetection`) that helps to identify where the
 or whether Reactor Netty already forwarded the ownership of the buffers to the application/framework.
 By default, this logger is disabled. To enable it, increase the log level to `DEBUG`.
 
+Another way to detect memory leaks is to monitor _reactor.netty.bytebuf.allocator.active.heap.memory_ and _reactor.netty.bytebuf.allocator.active.direct.memory_ meters:
+
+- The _reactor.netty.bytebuf.allocator.active.heap.memory_ provides the actual bytes consumed by in-use buffers allocated from heap buffer pools
+- The _reactor.netty.bytebuf.allocator.active.direct.memory_ provides the actual bytes consumed by in-use buffers allocated from direct buffer pools
```

Comment:
It is more visible if we use `something` instead of _something_


```suggestion
Another way to detect memory leaks is to monitor `reactor.netty.bytebuf.allocator.active.heap.memory` and `reactor.netty.bytebuf.allocator.active.direct.memory` meters:

- The `reactor.netty.bytebuf.allocator.active.heap.memory` provides the actual bytes consumed by in-use buffers allocated from heap buffer pools
- The `reactor.netty.bytebuf.allocator.active.direct.memory` provides the actual bytes consumed by in-use buffers allocated from direct buffer pools
```